articles/113_ros_time.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Clock and Time
+permalink: articles/clock_and_time.html
+abstract:
+  This article describes the ROS primatives to support programming which can run both in realtime as well as simulated time.
```

Comment:
You need to define "real time" here. I assume that you're talking about running in "world time" (I don't actually know if there's an agreed-upon phrase to describe that), not meeting deadlines.